src/main/java/seedu/address/model/parcel/Parcel.java

```diff
@@ -187,4 +187,13 @@ public String toString() {
         return getAsText();
     }
 
+    @Override
+    public int compareTo(Object o) {
+        Parcel other = (Parcel) o;
+        if (other == this) {
```

Comment:
Why does this other == this matter? oO (either way, if its the same object, compareTo would have done its job

Reply:
Just to short-circuit it early I guess. Idk seems like the convention to check for this when I learnt it :/